packages/frontend/src/components/UnirepEvent.jsx

```diff
@@ -3,14 +3,16 @@ import { Link } from "react-router-dom";
 import './eventCard.css'
 
 
-const UnirepEvent = ({ address, appName, epoch, user, repGiven })  => {
+const UnirepEvent = ({ address, current, users, reputation, nextEpoch })  => {
```

Comment:
Most of these props (all of them?) should be loaded from the store or calculated, not passed as values. I might be wrong because i'm not looking at the logic closely, but it's a feeling.